### PR TITLE
Add ability to use custom font

### PR DESCRIPTION
**Pull request by chasseurmic**
I made some additions to the library in order to give the user the ability to use their own custom fonts.

### CODE REVIEW

File: Classes/YHRoundBorderedButton.m [hunk 1]
```diff
@@ -43,12 +53,26 @@ - (id)initWithFrame:(CGRect)frame
...
+}
+
 - (void)setup
 {
     [self setTitleColor:[self tintColor] forState:UIControlStateNormal];
-    [self setTitleColor:[UIColor whiteColor] forState:UIControlStateHighlighted];
     [self setTitleColor:[UIColor grayColor] forState:UIControlStateDisabled];
-    [self.titleLabel setFont:[UIFont boldSystemFontOfSize:13]];
+    if (_userFont) {
+        [self setTitleColor:[UIColor blackColor] forState:UIControlStateHighlighted];
```
Comment (yhpark) [inline Classes/YHRoundBorderedButton.m]: I'm using `self tintColor` for borders, fonts or any other uses. Please replace black color with `self tintColor`.

**Comment by chasseurmic** [comment]
Ok, here we go!

**Comment by yhpark** [comment]
@chasseurmic Seems you checked in idea files. Could you take them out with amend commit? 
Thanks

**Comment by chasseurmic** [comment]
Sorry about that. Amended.

**Pull request closed without merging**